specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-09-01-preview/examples/ApiManagementListWorkspaceLinks.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters": {
+    "resourceGroupName": "rg1",
+    "api-version": "2023-09-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "serviceName": "service1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
```

Comment:
The ProxyResource(v5) type has also systemData defined

Reply:
We dont return system data for this type but I was told to use latest definition. hence used the latest but since we dont return system data wont be returning in example

Reply:
ApiManagementWorkspaceLinksResource refers to type V2